recipes/octave/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib -Wl,-rpath,${PREFIX}/lib ${LDFLAGS}"
+
+chmod +x configure
+chmod +x build-aux/mk-opts.pl
+
+./configure --help
+./configure --prefix=$PREFIX \
+    --disable-docs \
+    --enable-readline \
+    --enable-shared \
+    --with-lapack="-lopenblas" \
+    --enable-dl \
+    --without-qhull \
+    --without-qrupdate \
+    --with-qt=4 \
```

Comment:
No problem. I actually prefer to have a build number with `qt4` first and then we can update to `qt5` later. I don't want to maintain a `qt4` version of all the packages though, but it is nice to release at least one.

PS: we should open issues on both `gnuplot` and `qscintilla2` to upgrade to `qt5` whenever it is possible.